### PR TITLE
Env configured port

### PR DESCRIPTION
**Pull request by DanielElisenberg**
# ENV CONFIGURED PORT

Can configure the port the wsgi webserver runs on through the environment, and smaller cleanups.

### What's changed?
* Turns out we don't need to change the way we run the gunicorn server. When you run gunicorn it first looks to the arguments for config, then to the environment, then to the defaults. So if we do not explicitly name a port for it to listen to it will instead run on the port specified by the PORT environment variable. (The same applies to UVICORN except for the name of the ENV var being UVICORN_PORT). Simply removed this argument from the docker CMD. Will update the docker compose repository accordingly.
* Smaller clean ups after looking through the code

_Tested all changes with the datastore-testing repository_

### CODE REVIEW

File: metadata_service/exceptions/exceptions.py [hunk 1]
```diff
@@ -3,7 +3,21 @@ def __init__(self, msg):
         super().__init__()
         self.message = {
             'type': 'DATA_NOT_FOUND',
-            'code': 105,
+            'code': 404,
```
Comment (pawbu) [inline metadata_service/exceptions/exceptions.py]: This has to be 105 as it is specified in the contract here: https://gitlab.sikt.no/raird/data-store-api-doc#error-responses

File: metadata_service/exceptions/exceptions.py [hunk 2]
```diff
@@ -3,7 +3,21 @@ def __init__(self, msg):
...
+
+    def to_dict(self):
+        return self.message
+
+
+class InvalidStorageFormatException(Exception):
+    def __init__(self, msg):
+        super().__init__()
+        self.message = {
+            'type': 'DATA_NOT_FOUND',
```
Comment (pawbu) [inline metadata_service/exceptions/exceptions.py]: Maybe use HTTP 500 with
```
    type: "SYSTEM_ERROR"
    code: 202
```

File: tests/unit/domain/test_metadata.py [hunk 3]
```diff
@@ -1,29 +1,35 @@
...
+
 from metadata_service.adapter import datastore
 from metadata_service.domain import metadata
-from metadata_service.exceptions.exceptions import DataNotFoundException
+from metadata_service.exceptions.exceptions import (
+    InvalidStorageFormatException
+)
 
+
+FIXTURES_DIR = 'tests/resources/fixtures'
```
Comment (pawbu) [inline tests/unit/domain/test_metadata.py:13]: 💯

File: metadata_service/domain/metadata.py [hunk 4]
```diff
@@ -75,31 +76,35 @@ def find_languages():
...
+    return metadata_all
+
+
+def _clear_code_list_and_missing_values(data_structures: list[dict]):
+    represented_variables = []
+    for metadata in data_structures:
+        represented_measure = (
+            metadata['measureVariable']['representedVariables']
+        )
+        represented_identifiers = list(chain(*[
```
Comment (pawbu) [inline metadata_service/domain/metadata.py:93]: 🐍

File: metadata_service/exceptions/exceptions.py (hunk 1 above)
Comment (DanielElisenberg) [reply metadata_service/exceptions/exceptions.py]: was not aware of this! Will fix 👍🏻

**Review by pawbu — APPROVED** [review]
Looks good!

**Pull request merged**